client/src/ActionsAndReducers/dbWargames/wargames_Reducer.ts

```diff
@@ -57,9 +57,7 @@ export const wargamesReducer = (state = initialState, action: WargameActionTypes
 
   if (handler) {
     handler(newState, action, tab)
-  } else {
-    console.warn(`Heandler not defined for a action type: ${action.type}`)
-  }
```

Comment:
Hello @lilitkarapetyan - are you sure that we don't need handlers for these events?

Are they being handled somewhere else?  If so, would it be wise to refer to the handler from this file?

![image](https://github.com/serge-web/serge/assets/1108513/7bc38128-e42c-437d-9eb9-7f89c3e5cb4c)

Reply:
Yes, the handlers for these events are handled in a different part of the codebase.